src/lsstseries/analysis/structure_function_calculators/basic_structure_function_calculator.py

```diff
@@ -0,0 +1,126 @@
+from typing import List
+
+import numpy as np
+from scipy.stats import binned_statistic
+
+from lsstseries.analysis.structure_function_argument_containers.structure_function_argument_container import (
```

Comment:
Yikes! That's quite the import!

I'm wondering a different directory structure would be helpful, both for reducing the path lengths and keeping config and implementation closer:

```
_ analysis
  | _ structure_function
     | _ base
     | _ bauer2009
     |  | _ README ## includes more information on formula, citation, etc
     |  | _ container.py
     |  | _ calculator.py
     | _ kozlowski16
     | _ sf2
        | _ README 
        | _ etc.
```

Reply:
I've rearranged the directory structure a bit. In some cases it made the import lines shorter, in others it resulted in a few extra characters. 

However, I believe that this will result in more organized code going forward, Thanks for calling this out!